tests/test_sides.py

```diff
@@ -0,0 +1,20 @@
+# Tests for functionality on part sides.
+
+from cycax.cycad import SheetMetal
+from cycax.cycad.location import BACK, BOTTOM, FRONT, LEFT, RIGHT, TOP
+
+sides = (LEFT, RIGHT, TOP, BOTTOM, FRONT, BACK)
```

Comment:
I see lists of all the sides often, maybe we should move it to cycax.cycad.location.
SIDES = (LEFT, RIGHT, TOP, BOTTOM, FRONT, BACK)

Reply:
SS179